jdaviz/configs/mosviz/plugins/viewers.py

```diff
@@ -105,6 +214,22 @@ def set_plot_axes(self):
         self.figure.axes[1].label_offset = "-50"
 
 
+@viewer_registry("mosviz-profile-viewer", label="Profile 1D")
+class MosvizProfileView(SpecvizProfileView):
+    # categories: zoom resets, zoom, pan, subset, select tools, shortcuts
+    tools_nested = [
+                    ['mosviz:homezoom', 'jdaviz:prevzoom'],
+                    ['mosviz:boxzoom', 'mosviz:xrangezoom', 'jdaviz:yrangezoom'],  # noqa
+                    ['mosviz:panzoom', 'mosviz:panzoom_x', 'jdaviz:panzoom_y'],  # noqa
+                    ['bqplot:xrange'],
+                    ['jdaviz:selectline'],
+                    ['jdaviz:sidebar_plot', 'jdaviz:sidebar_export']
+                ]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Is this necessary since it is not really doing anything extra in the constructor?
```suggestion
```